docs/pages/reference/architecture/agent-update-management.mdx

```diff
@@ -79,11 +79,11 @@ latest agent version.
 
 ### Self-hosted Teleport
 
-Self-hosted Teleport users can set up automatic agent updates. They must host
-their version server and choose their target version. They are responsible for
-ensuring the targeted version is compatible with their current auth/proxy
-versions. They must also monitor the agent's health and rollout status to
-ensure every agent is healthy and running the correct version.
+Self-hosted Teleport Enterprise users can set up automatic agent updates.
```

Comment:
I think we should refer to the reader as "you" instead of "they" - _if you are self-hosting Teleport then you must..._

Reply:
Thanks @zmb3 , please see update.